File: Backend go/domain/entity/salereport.go
package entity

import "time"

type Salereport struct {
	SaleId           int       `json:"saleid"`
	CustomerName     string    `json:"customername"`
	CustomerDocument string    `json: "customerdoccument"`
	SallerName       string    `json: "sallername"`
	TotalAmount      float32   `json: "totalamount"`
	DiscountAmount   float32   `json: "discountamount"`
	FinalAmout       float32   `json: "finalamout"`
	SaleDate         time.Time `json: "saledate"`
	SaleTime         string    `json: "saletime`
	PaymentMethod    string    `json: "paymentmethod"`
	SaleNotes        string    `json: "salenotes"`
}

File: Front_Blazor/KomercioPlus/Model/Entity/SaleReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.Text.Json.Serialization;


namespace KomercioPlus.Model.Entity
{
    public class SaleReport
    {
         [JsonPropertyName("saleid")]
        public int SaleId { get; set; }

        [JsonPropertyName("customername")]
        public string CustomerName { get; set; }

        [JsonPropertyName("CustomerDocument")]
        public string CustomerDocument { get; set; }

        [JsonPropertyName("SallerName")]
        public string SallerName { get; set; }

        [JsonPropertyName("TotalAmount")]
        public decimal TotalAmount { get; set; }

        [JsonPropertyName("DiscountAmount")]
        public decimal DiscountAmount { get; set; }

        [JsonPropertyName("FinalAmout")]
        public decimal FinalAmount { get; set; }

        [JsonPropertyName("SaleDate")]
        public DateTime SaleDate { get; set; }

        // Vem como "12:11:05.000000"
        [JsonPropertyName("SaleTime")]
        public string SaleTime { get; set; }

        [JsonPropertyName("PaymentMethod")]
        public string PaymentMethod { get; set; }

        [JsonPropertyName("SaleNotes")]
        public string SaleNotes { get; set; }
    }
}